src/config.ts

```diff
@@ -57,14 +57,20 @@ import { LoginOptions, DeepPartial } from './auth0-session';
  * ### 2. Create your own instance using {@link InitAuth0}
  *
  * If you don't want to configure the SDK with environment variables or you want more fine grained control over the
- * instance, you can create an instance yourself and use the handlers and helpers from that, eg:
+ * instance, you can create an instance yourself and use the handlers and helpers from that.
+ *
+ * First, export your configured instance from another module:
  *
  * ```js
  * // utils/auth0.js
  * import { initAuth0 } from '@auth0/nextjs-auth0';
  *
  * export default initAuth0({ ...ConfigParameters... });
+ * ```
+ *
+ * Then import it into your route handler:
  *
+ * ```js
```

Comment:
As the code snippet actually deals with code in two files, it's much clearer to have these as two snippets. I just added some intro text to each for context.